sdk/src/main/scala/com/horizen/block/SidechainCommitmentTree.scala

```diff
@@ -78,8 +78,8 @@ class SidechainCommitmentTree {
       btrList.toArray,
       certificate.customFieldsOpt(version).asJava,
       certificate.endCumulativeScTxCommitmentTreeRoot,
+      certificate.ftMinAmount, //TODO: temporal dirty porkaround until SC zendoo_commitment_tree_add_cert is fixed
```

Comment:
let's add a bit more info about the porkaround:
that we swapped the order of the `ftMinAmount` and `btrFee` and now setting them in the wrong order to be consistent with the wrong order on MC side

Reply:
improved todo comment